### PR TITLE
Remove hash from ActivityStreams profile URL

### PR DESCRIPTION
**Pull request by strugee**
Thanks to saranix on IRC for catching this.

### CODE REVIEW

**Comment by clacke** [comment]
EDIT: Never mind me, this was about the profile, not the namespace.
EDIT2: Actually, maybe correct the commit message to reflect that.

----

No, that's incorrect.

From https://www.w3.org/ns/activitystreams : 
> as:
> http://www.w3.org/ns/activitystreams#

The `#` is important. Removing it makes the namespace a different one. The reason it's there is because in RDF (as opposed to in XML, where they're a pair), identifiers within namespaces are represented simply as URIs made from the concatenation namespace string + name string, and https://www.w3.org/ns/activitystreams#Activity looks so much better than https://www.w3.org/ns/activitystreamsActivity.

https://www.w3.org/TR/activitystreams-vocabulary/#dfn-activity shows the URI to be https://www.w3.org/ns/activitystreams#Activity .

**Comment by strugee** [comment]
Okay, I definitely don't know RDF enough to get what you just said ;)

Is this a problem or not? Either way *something* is wrong since the rest of the document uses the hashless form, so there's inconsistency.

**Comment by clacke** [comment]
TL;DR: Carry on, I'm just making a fool of myself. I'm sorry for the confusion. You're doing the right thing. But do change `s/namespace/profile/` in the commit message, because you confused me first. ;-)

----

Your commit message said you're changing the namespace, and you're not. You're changing the profile reference, which is a different thing.

The namespace has a trailing hash. The profile, which is a request for a certain profile (format) of JSON-LD, does not have a trailing hash, as defined in https://www.w3.org/TR/activitystreams-core/#media-type .

**Comment by strugee** [comment]
@clacke just pushed an updated commit message.
